lib/model_2_factory/model_2_factory.rb

```diff
@@ -72,7 +73,14 @@ def self.factory_suffix
     end
   end
 
+  def self.create_factory_dir(dirname)
+    unless File.directory?(dirname)
```

Comment:
__[rubocop]__
* Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. [:ref](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)

<!-- Comment by [SideCI](https://sideci.com) -->